js/src/hooks/useGoogleAdsAccountBillingStatus.js

```diff
@@ -8,14 +8,18 @@ import { useSelect } from '@wordpress/data';
  */
 import { STORE_KEY } from '.~/data/constants';
 
+const selectorName = 'getGoogleAdsAccountBillingStatus';
+
 const useGoogleAdsAccountBillingStatus = () => {
 	return useSelect( ( select ) => {
-		const billingStatus = select(
-			STORE_KEY
-		).getGoogleAdsAccountBillingStatus();
+		const selector = select( STORE_KEY );
 
 		return {
-			billingStatus,
+			billingStatus: selector[ selectorName ](),
+			hasFinishedResolution: selector.hasFinishedResolution(
+				selectorName,
+				[]
+			),
```

Comment:
❓  What is exactly this change for? I guess it's for refactor `selectorName`  and not write twice `getGoogleAdsAccountBillingStatus` but I'm not sure

Reply:
> What is exactly this change for?

To add the `hasFinishedResolution` property to the returning object. It's used [here](https://github.com/woocommerce/google-listings-and-ads/blob/ae4682c0f68206f73f36ac70dbad83e0d41cb56d/js/src/components/paid-ads/billing-card/billing-card.js#L25-L32).

> I guess it's for refactor `selectorName` and not write twice `getGoogleAdsAccountBillingStatus` but I'm not sure

Creating a variable for the selector name was based on a past code review https://github.com/woocommerce/google-listings-and-ads/pull/377#discussion_r601730266 to prevent from receiving the same suggestion. In this case, while I thought/think both ways are fine, and there is no consistency in the current codebase for similar hooks such as [useGoogleMCAccount](https://github.com/woocommerce/google-listings-and-ads/blob/ae4682c0f68206f73f36ac70dbad83e0d41cb56d/js/src/hooks/useGoogleMCAccount.js#L37-L41), I don't have an obvious preference. So, I'm happy to adjust if you have a more obvious preference for it.

Reply:
Nono, It's ok. Was just curious if was just for that variable name or something more. Im fine also for both.